trtools/utils/tests/test_trharmonizer.py

```diff
@@ -182,24 +220,25 @@ def test_TRRecord_full_alleles():
     alt_alleles = []
     for allele in full_alts:
         alt_alleles.append(allele[1:-1])
-    motif = 'FOO'
-    ID = 'BAR'
+    motif = 'CAG'
+    ID = 'STR1'
+    test_rec = DummyCyvcf2Record(None, full_ref, full_alts)
 
     with pytest.raises(ValueError):
-        trh.TRRecord(dummy_record1, None, None, motif, ID, None,
+        trh.TRRecord(test_rec, None, None, motif, ID, None,
```

Comment:
I think it's not super clear what is being tested here, maybe a comment could help.

Reply:
*A comment for each assert or raises instance

Reply:
Yeah, I've no clue what I was trying to test 😅 so you're definitely right it should be commented. I'll reverse engineer the comments I guess.